src/components/EReceipt.tsx

```diff
@@ -50,11 +47,11 @@ function EReceipt({transactionID, transactionItem, isThumbnail = false}: EReceip
         created: transactionDate,
         cardID: transactionCardID,
         cardName: transactionCardName,
-    } = getTransactionDetails(transactionItem ?? transaction, CONST.DATE.MONTH_DAY_YEAR_FORMAT) ?? {};
+    } = transactionDetails ?? {};
```

Comment:
eg. looking at the trace half the time is spent on https://github.com/Expensify/App/blob/821f714f97c8a37ea50d8bb19667e1c539b071cc/src/libs/ReportUtils.ts#L3995 which is not being used here

Reply:
that's 25ms though and requires rewriting to separate utils inside of the component/useEReceipt